tcms/testplans/templates/testplans/get.html

```diff
@@ -89,7 +89,94 @@ <h2 class="card-pf-title">
                     </h2>
                 </div>
                 <div class="card-pf-body">
-                    <div id="confirmed-testcases" class="list-group list-view-pf list-view-pf-view"></div>
+                    <div class="container-fluid">
+                        <div class="row toolbar-pf">
+                            <div class="col-sm-12">
+                                <form class="toolbar-pf-actions">
+                                    <div class="form-group">
+                                        <input type="checkbox">
+                                    </div>
+                                    <div class="form-group">
```

Comment:
![Screenshot_20200804_234118](https://user-images.githubusercontent.com/8727922/89342981-40e9bb00-d6ac-11ea-88b2-59d4084210f1.png)
![Screenshot_20200804_234137](https://user-images.githubusercontent.com/8727922/89342986-41825180-d6ac-11ea-9380-f5c0e6fd7e81.png)
This class only add padding-left, IMO we dont need it.

Reply:
The code pattern from upstream uses it so we should follow as close as possible and stay consistent. IDK what might change in the future and decide to use this class then we'll end up with a hard to debug problem if this happens. 

Toolbar in TR page redesign also uses it so we need the consistency here.

Reply:
Yes the toolbar in the TR page use it, but the filter is the first child element there. The examples in patternfly also use the filter as first element of the toolbar. I have to use some custom styling to fix this(added as inline style at the moment).

Reply:
I don't understand how the position of the toolbar relates to whether or not we follow the examples from upstream? Please explain. 

> Yes the toolbar in the TR page use it, but the filter is the first child element there.

Note: that page is not fully implemented and is missing the check-all checkbox.

> Yes the toolbar in the TR page use it, but the filter is the first child element there.

From what I can see your inline style refers to the check-all checkbox and not the form-group which contains the actual filtering widget. Also I see the HTML has been updated - will check out how it looks.

Reply:
The root cause of the problem is that the right border of the form-group(which contains the check-all checkbox) is next to the filter button. There is no space between them, thats why I added inline style to the checkbox group. The examples does not include extra elements before filter button.

Reply:
This is unrelated to the missing `toolbar-pf-filter` class, isn't it ?

Reply:
If I add it the elements go next to each other, thats why I add the inline style. If I drop the toolbar-pf-filter class everything looks good.

Reply:
Now I see what you mean. However let's keep this `toolbar-pf-filter` class b/c the inline style above needs a few more adjustments:

1) Make the margin 15px so that the left-hand side of the filter widget aligns vertically with the TC text below. The rows in the table have margin of 15px between the checkbox border and the text anyaway
2) add padding-right: 15px which will align the check-all border vertically with all of the checbox borders below it.